demos/fleet_provisioning/fleet_provisioning_with_csr/pkcs11_operations.c

```diff
@@ -489,7 +500,7 @@ static CK_RV provisionPrivateECKey( CK_SESSION_HANDLE session,
         {
             { CKA_CLASS,     NULL /* &privateKeyClass*/, sizeof( CK_OBJECT_CLASS )                     },
             { CKA_KEY_TYPE,  NULL /* &privateKeyType*/,  sizeof( CK_KEY_TYPE )                         },
-            { CKA_LABEL,     label,                      ( CK_ULONG ) strlen( ( const char * ) label ) },
+            { CKA_LABEL,     ( void * ) label,           ( CK_ULONG ) strlen( ( const char * ) label ) },
```

Comment:
The explicit cast to `const char *` shouldn;t be required

Reply:
It is as we are losing const here. Would work without cast if the PKCS #11 API took const void * instead of void *, but this is not the case.